week06_policy_based/atari_wrappers.py

```diff
@@ -261,6 +262,7 @@ def step(self, action):
             self.rewards[i] = 0
 
```

Comment:
`self.global_step += self.nenvs`

Reply:
Ok, but then there should exist self.nenvs, so here should be `self.nenvs = ...`
https://github.com/yandexdataschool/Practical_RL/blob/master/week06_policy_based/atari_wrappers.py#L216

Reply:
Right! It's better to initialize it at construction though, or just get from the shape of self.rewards for example :)